BraveShared/Extensions/UIStackViewExtensions.swift

```diff
@@ -27,4 +27,14 @@ extension UIStackView {
       }
     }
   }
+  // Adds Background StackView with Color and Corner Radius
+  public func addBackground(color: UIColor, cornerRadius: CGFloat? = nil) {
```

Comment:
To be honest I dont think this really belongs in an extension, it should just be done in each piece of UI that will actually define and handle it (i.e. the parent of the stack view should be the one setting the background color + corner radius.

If that piece of UI is repeated multiple times in the same configuration then its possible you'd want to subclass the view itself

Reply:
The extension is removed and  subclass is added as your suggestion.